.gitignore

```
@@ -9,3 +9,4 @@ node_modules
 
 /wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/.openapi-generator
 /wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/.openapi-generator-ignore
+/wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/git_push.sh
```

Nach der Generierung hatte ich noch 2 weitere unversionierte Files

```suggestion
/wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/git_push.sh
/wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/.gitignore
/wls-gui-wahllokalsystem/src/api/wls-clients/generated-*-api/.npmignore
```

---

die habe ich vergessen, danke. siehe [9ca4ff0](https://github.com/it-at-m/Wahllokalsystem/pull/763/commits/9ca4ff000db8c184a805785e4f62cf57f590c2cf)